web/src/app/wizard/WizardForm.js

```diff
@@ -57,7 +57,9 @@ export default class WizardForm extends React.PureComponent {
             <StepIcon icon='1' />
           </Grid>
           <Grid item xs={10}>
-            <Typography variant='h6'>Team Details</Typography>
+            <Typography variant='h6' variantMapping={{ h6: 'h2' }}>
```

Comment:
Can we have a quick group discussion about the variantMapping?

I feel like there needs to be a better way than having to specify this everywhere. Also, it's not obvious what it should be set too for future work that others will do.